### PR TITLE
Add createRequestContext and createRenderContext functions

### PR DESCRIPTION
**Pull request by ganemone**
These functions can be useful when trying to test middleware
or APIs which rely on a context object. Usage example:

```js
import {createRequestContext, createRenderContext} from 'fusion-test-utils';
const ctx = createRenderContext('/')
const ctx2 = createRequestContext('/', {method: 'POST'});
// ... etc
```

### CODE REVIEW

File: flow-fixtures/test.js [hunk 1]
```diff
@@ -15,4 +20,13 @@ test('Some content', async assert => {
   const simulator = getSimulator(app);
   const a: string = simulator.getService(TokenA);
   assert.equal(a, 'test');
+  createRequestContext('/');
+  createRenderContext('/');
+  createRequestContext('/', {
```
Comment (AlexMSmithCA) [inline flow-fixtures/test.js]: Should we add a test to ensure that the options are passed through to the request correctly?

File: README.md [hunk 2]
```diff
@@ -71,6 +71,20 @@ const sim = getSimulator(app);
 const logger = sim.getService(LoggerToken);
 ```
 
+#### `createRequestContext(url: String, options: ?Object) => ctx`
```
Comment (AlexMSmithCA) [inline README.md]: Minor: let's use the `Context` in place of `ctx` here for consistency.

File: flow-fixtures/test.js (hunk 1 above)
Comment (ganemone) [reply flow-fixtures/test.js]: Added tests

**Comment by ganemone** [comment]
!merge

**Pull request merged**